products/godot.md

```diff
@@ -17,16 +17,23 @@ auto:
     regex: ^(?<version>\d+(\.\d+){1,3})-stable$
     template: "{{version}}"
 releases:
--   releaseCycle: "3.4"
+-   releaseCycle: "3.5"
     support: true
     eol: false
+    latest: "3.5"
+    link: https://godotengine.org/article/godot-3-5-cant-stop-wont-stop
+    latestReleaseDate: 2022-08-05
+    releaseDate: 2022-08-05
+-   releaseCycle: "3.4"
+    support: false
+    eol: false
     latest: "3.4.5"
-    link: https://godotengine.org/article/maintenance-release-godot-3-4-2
+    link: https://godotengine.org/article/maintenance-release-godot-3-4-5
```

Comment:
do we need link ? i mean if you remove this changelogTemplate probably will place correct link

Reply:
I just tested and indeed this link is not needed